examples/experiment.py

```diff
@@ -10,7 +10,7 @@ def main():
     device = 'cpu'
     timesteps = 40000
     run_experiment(
-        [dqn(), a2c()],
+        [dqn.hyperparameters(), a2c.hyperparameters()],
```

Comment:
This can just be:

`[dqn, a2c]`

Or, arguably:

`[dqn.device('cuda'), a2c.device('cuda')]`

Reply:
Actually on further contemplation, I think my original way is more helpful for people trying to use this as an example

Reply:
It's not really correct usage though. Maybe it would be better to show a few different ways of using the builder object, e.g. `[dqn, dqn.hyperparameters(lr=1e2), a2c`], custom models, etc.

Reply:
done